jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/db2/DB2SqlDialect.java

```diff
@@ -107,4 +114,11 @@ public NullSorting getDefaultNullSorting() {
     public String getStringLiteral(final String value) {
         return "'" + value.replace("'", "''") + "'";
     }
+
+    @Override
+    public void validateProperties() throws PropertyValidationException {
+        super.validateDialectName(getPublicName());
+        super.validateSupportedPropertiesList(supportedProperties);
```

Comment:
Add an abstract method `getSupportedProperties()` in `AbstractSqlDialect` and use that in `validateProperties()`.
Then: remove this line.
Repeat for all dialects.

Reply:
done